pages/version/1.markdown

```diff
@@ -175,7 +175,7 @@ In future versions, defined keys will always adhere to these rules:
 
 ## Suggestions for Publishers
 
-JSON Feed files must be served using the same MIME type — `application/json` — that’s used whenever JSON is served.
+JSON Feed files must be served using the MIME type — `application/feed+json`.
```

Comment:
The `—` should also be removed. In the current sentence it doesn’t make much sense/read well.